features/_your_results/contributions_above_maximum_message.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Limit Tax Relief
+  In order to get a clear picture of the impact on my take-home pay.
+  As an employee contributing greater than £40,000 per year to my pension
+  I want the results to reflect the fact that I will not get tax relief
+  for any contribution over £40,000 per year.
+
+  Background:
+    Given I am on the Your Details step
+
+    Scenario Outline: Employee contributing more than the equivalent of £40,000 per year to pension
+      And I am a "25" year old "<gender>"
+      And my salary is "<salary>" "<salary_frequency>" with "Full" contribution
+      And I click the Next button
+      And my contribution is "60" percent
```

Comment:
in my opinion you could probably collapse the number of steps detailed here to make it less verbose and move some of these steps into the step definitions... you probably don't even need the background step
```
Given I have valid details
And my salary is "<salary" "<salary_frequency>"
When my contribution is "60" percent
And I move onto the results page
Then I should see the contribution above maximum "<message>"
```

Reply:
You could even move the welsh and english message translations into the back end as well perhaps.

Reply:
`Given I have valid details` includes the step `step %{I choose to make minimum contributions}`. I need contributions to be `full`.

Also have you tried that - because all the tests failed when I did.

I'm not sure I want to create more step definitions as that file is quite verbose atm and has alot of steps which look very similar.

Reply:
and you can't make use of any of the current steps? 

`Given I have valid details` can be used to setup a default user with the details that are inconsequential, you can then update or change those values in subsequent steps if needed. I haven't looked if there is a better step for you to use yet, there may be. I just put it in as a suggestion. Admittedly this may add a small amount of time to running the tests.

I'm happy to leave the first steps and not use the `valid details`. I think the two main bits I'd simplify would be 

- joining the
```
And I click the Next button
And my contribution is "60" percent
```
- and moving the gender out of the examples steps.

Reply:
@willhall88 I've refactored the feature, lmk what you think.